Source/WebKit/UIProcess/API/glib/WebKitInstallMissingMediaPluginsPermissionRequest.cpp

```diff
@@ -132,15 +100,6 @@ static void webkit_install_missing_media_plugins_permission_request_class_init(W
     objectClass->dispose = webkitInstallMissingMediaPluginsPermissionRequestDispose;
```

Comment:
You can get rid of the dispose func now, since it just calls deny and that doesn't do anything anymore.